regression/contracts/assigns_enforce_arrays_06/main.c

```diff
@@ -0,0 +1,12 @@
+void assign_out_under(int a[], int len) __CPROVER_assigns(a[8])
+{
+  a[7] = 5;
```

Comment:
It's quite similar. In this case we cant to see if we report failure correctly when the specification doesn't have a range, but a single cell (i.e., `a[8]` instead of `a[2, 5]`).